CMakeLists_files.cmake

```diff
@@ -31,6 +31,10 @@
 list (APPEND TEST_SOURCE_FILES
 	)
 
+list(APPEND MAIN_SOURCE_FILES
+	opm/upscaling/RelPermUtils.cpp
```

Comment:
On 14/09/15 10:37, Atgeirr Flø Rasmussen wrote:
>
> In CMakeLists_files.cmake 
> <https://github.com/OPM/opm-upscaling/pull/153#discussion_r39371057>:
>
> > @@ -31,6 +31,10 @@
> >  list (APPEND TEST_SOURCE_FILES
> >  	)
> >
> > +list(APPEND MAIN_SOURCE_FILES
> > +	opm/upscaling/RelPermUtils.cpp
>
> Should not RelPermUtils.hpp also be added somewhere?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OPM/opm-upscaling/pull/153/files#r39371057>.
>
don't think so?

it's application helpers. not really relevant for use outside those apps.